### ISSUE TITLE
오탈자 및 수정 제안

### ISSUE THREAD

**Issue by codingbowoo** [9bow/PyTorch-tutorials-kr#81]
## 문서 URL
- **URL**: https://tutorials.pytorch.kr/beginner/blitz/neural_networks_tutorial.html

## 변경 사항
(1) 신경망의 가중치를 갱신합니다. 일반적으로 다음과 같은 간단한 규칙을 사용합니다: 가중치(**wiehgt**) = 가중치(weight) - 학습율(learning rate) * 변화도(gradient)
(2) 신경망의 가중치를 갱신합니다. 일반적으로 다음과 같은 간단한 규칙을 사용합니다: _새로운_ 가중치(**weight**) = _기존_ 가중치(weight) - 학습율(learning rate) * 변화도(gradient)
* * *
If you have a single sample, just use input.unsqueeze(0) to add a fake batch dimension.
(1) 만약 하나의 샘플만 있다면, input.unsqueeze(0) 을 사용해서 _가짜_ 차원을 추가합니다. 
(2) 만약 하나의 샘플만 있다면, input.unsqueeze(0) 을 사용해서 _가상의_ 차원을 추가합니다.

## 추가 정보
1. 설명을 돕기 위해 "새로운", "기존"을 추가하면 좋을 것 같다고 생각했습니다. 1번 오탈자 및 공식은 해당 문서에 2번 나와서, 두 경우 모두 수정이 필요합니다. 
2. 원문이 "fake"이긴 하나 "가짜 차원" 이라는 말이 어색하게 여겨져 수정을 제안합니다.

\+ 해당 문서 내 코드의 주석 번역이 필요합니다.

**Comment by 9bow** [comment]
제안 감사합니다. :)
PR을 만들어주실 수 있으실까요?

**Comment by 9bow** [comment]
@itsnamgyu 님, 혹시 다른 의견 있으시면 부탁드립니다~
